packages/config/config/devices/0x0258/nas-pd02z-2.json

```diff
@@ -0,0 +1,261 @@
+			"defaultValue": 1,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "0.5 seconds",
+					"value": 0
+				},
+				{
+					"label": "1 second",
+					"value": 1
+				},
+				{
+					"label": "1.5 seconds",
+					"value": 2
+				},
+				{
+					"label": "2 seconds",
+					"value": 3
+				},
+				{
+					"label": "2.5 seconds",
+					"value": 4
+				},
+				{
+					"label": "3 seconds",
+					"value": 5
+				},
+				{
+					"label": "3.5 seconds",
+					"value": 6
+				},
+				{
+					"label": "4 seconds",
+					"value": 7
+				},
+				{
+					"label": "4.5 seconds",
+					"value": 8
+				},
+				{
+					"label": "5 seconds",
+					"value": 9
+				},
+				{
+					"label": "5.5 seconds",
+					"value": 10
+				},
+				{
+					"label": "6 seconds",
+					"value": 11
+				},
+				{
+					"label": "6.5 seconds",
+					"value": 12
+				},
+				{
+					"label": "7 seconds",
+					"value": 13
+				},
+				{
+					"label": "7.5 seconds",
+					"value": 14
+				},
+				{
+					"label": "8 seconds",
+					"value": 15
+				}
+			]
+		},
+		{
+			"#": "5",
+			"label": "Motion Sensor",
+			"$import": "~/templates/master_template.json#base_enable_disable",
+			"defaultValue": 1
+		},
+		{
+			"#": "6",
+			"label": "Motion Sensor: Clear Time",
+			"description": "Time in seconds to wait before sending a 'Clear motion detected' message after the last motion was detected. This determines the minimum time the sensor remains on after motion was detected. Possible values are 10 to 3600 seconds.",
+			"unit": "seconds",
+			"valueSize": 2,
+			"minValue": 10,
+			"maxValue": 3600,
+			"defaultValue": 30
+		},
+		{
+			"#": "7",
+			"$import": "templates/shenzhen_neo_template.json#blink_on_motion"
+		},
+		{
+			"#": "8",
+			"label": "Binary Sensor Report Enable",
+			"$import": "~/templates/master_template.json#base_enable_disable"
+		},
+		{
+			"#": "9",
+			"label": "Motion Sensor: Basic Set Level",
+			"description": "Value for BASIC_SET for nodes associated in Group 2. '100'- BASIC_SET = 0xFF (ON). '0'- BASIC_SET = 0x00 (OFF).",
+			"valueSize": 1,
+			"minValue": 1,
+			"maxValue": 100,
+			"defaultValue": 100
+		},
+		{
+			"#": "10",
+			"label": "Light Sensor: Measuring Interval",
+			"description": "Time interval for light sensor measurement of ambient light intensity. Higher values result in longer battery life.",
+			"unit": "seconds",
+			"valueSize": 2,
+			"minValue": 30,
+			"maxValue": 3600,
+			"defaultValue": 180
+		},
+		{
+			"#": "11",
+			"label": "Light Sensor: Intensity Differential Report",
+			"valueSize": 1,
+			"minValue": 1,
+			"maxValue": 127,
+			"defaultValue": 50
+		},
+		{
+			"#": "12",
+			"label": "Light Sensor: Intensity Threshold",
+			"description": "If the light intensity current measured is less than the settings, device will considered to be dark at current time. If configuration 13 is enabled and a motion event is detected, the device will send a BASIC_SET to the nodes that associated in group 2.",
```

Comment:
AFAIK other devices call this Night/Day Threshold. We should do the same here and update the corresponding parameters accordingly.